weave

```diff
@@ -1945,10 +1949,13 @@ EOF
     attach-router)
         check_running $CONTAINER_NAME
         enforce_docker_bridge_addr_assign_type
+        # We cannot use detect_awsvpc here, because HTTP server might not be started
+        ! docker inspect -f '{{.Config.Cmd}}' $CONTAINER_NAME | grep -q -- "--awsvpc" || AWSVPC=1
         create_bridge
         fetch_router_args
         setup_router_iface_$BRIDGE_TYPE
         wait_for_status $CONTAINER_NAME http_call $HTTP_ADDR
+        setup_awsvpc
```

Comment:
IMO the order of requests in the case of re-attaching the router seems to be a bit illogical:

1. `expose_ip` (in `setup_awsvpc`) requests the allocator to allocate a IP for `weave:expose`. The allocator returns the previously owned IP to `weave:expose` which gets assigned to the "weave" bridge afterwards. 
2. `populate_router` tries to reclaim the address returned in the (1) step.